testsuite/utils/pom.xml

```
@@ -271,6 +271,34 @@
                 </plugins>
             </build>
         </profile>
+        <profile>
+            <id>map-storage</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.codehaus.mojo</groupId>
+                        <artifactId>exec-maven-plugin</artifactId>
+                        <configuration>
+                            <systemProperties>
+                                <property><key>keycloak.profile.feature.map_storage</key><value>enabled</value></property>
```

```suggestion
                                <systemProperty><key>keycloak.profile.feature.map_storage</key><value>enabled</value></systemProperty>
```

Based on https://www.mojohaus.org/exec-maven-plugin/usage.html the `systemProperties` element should contain `systemProperty` elements. However, it works also with `property` so I am not sure this needs to be changed.

---

Thanks, good point.